src/Pound.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
+
+class Pound extends Component {
```

Comment:
You don't need these at the moment since you already have consts defined in App. Let's remove for now.